remote/templates/search.html

```diff
@@ -0,0 +1,15 @@
+<!-- wp:template-part {"slug":"header","tagName":"header"} /-->
+
+<!-- wp:group {"layout":{"inherit":true}} -->
+<div class="wp-block-group"><!-- wp:spacer {"height":"32px"} -->
+<div style="height:32px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+<!-- wp:search {"label":"Search","width":100,"widthUnit":"%","buttonText":"Search","style":{"border":{"radius":"0px"}}} /-->
+<!-- wp:spacer -->
+<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:pattern {"slug":"remote/posts-list"} /-->
+
+<!-- wp:template-part {"slug":"footer","tagName":"footer","className":"site-footer-container","layout":{"inherit":true}} /-->
```

Comment:
Do we need to add the `site-footer-container` className here?

Reply:
Nope, it seems like it is doing nothing so I'm removing it.